compat/src/suspense.js

```diff
@@ -11,7 +11,11 @@ options._catchError = function(error, newVNode, oldVNode) {
 		for (; (vnode = vnode._parent); ) {
 			if ((component = vnode._component) && component._childDidSuspend) {
 				// Don't call oldCatchError if we found a Suspense
-				return component._childDidSuspend(error, newVNode._component);
+				return component._childDidSuspend(
+					error,
+					newVNode._component,
+					oldVNode && oldVNode._hydrateDom && oldVNode._hydrating
```

Comment:
Just a thought (doesn't change the output or anything) - do you think we'll ever want to access the other properties of `oldVNode` from within `_childDidSuspend`?  If so we could pass `oldVNode` here and then check these properties in the _childDidSuspend implementation.  I don't have strong feelings either way, just occurred to me.

Reply:
done